### PR TITLE
[Draft] Enables respec darkmode

### LINKED ISSUE #2133: Insertions and deletions in PR diff preview are unreadable in dark mode
I’m not sure this is the proper home for this issue, but I just noticed that the color styles for the `ins` and `del` elements in our PR diff previews make their text unreadable if your OS environment is _in dark mode_.

A video capture demonstrating how the colors for ins and del elements in htmldiff’d W3C documents become unreadable in dark mode

I tracked this back to issue w3c/htmldiff-ui#12, where @tabatkins says:

> ...The answer is to make your documents darkmode-aware...

Is that something we can/should do for our documents?

### PR DESCRIPTION
**Pull request by daniel-montalvo**
Closes #2133

Enables darkmode compatibilty through respecConfig.darkMode

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/aria/pull/2138.html" title="Last updated on Jul 17, 2024, 10:00 AM UTC (284a549)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/2138/b6459d8...284a549.html" title="Last updated on Jul 17, 2024, 10:00 AM UTC (284a549)">Diff</a>

### CODE REVIEW

**Comment by daniel-montalvo** [comment]
This follows w3c/respec#3236

Much appreciated if anyone can please test this to see how it actually works in the ARIA context.

**Comment by pkra** [comment]
@daniel-montalvo this doesn't seem to have any any visual effect when switching my browser's dark/light mode. I also tried manually adding `?darkMode=true` to no avail.

**Comment by pkra** [comment]
Oh, as @adampage said, the diffs are (still) affected and still exhibiting the bug that Adam had reported.

**Comment by dontcallmedom** [comment]
The dark mode feature has been merged in respec, but not yet deployed in a release, so it is expected this would not have an effect yet

**Comment by daniel-montalvo** [comment]
> The dark mode feature has been merged in respec, but not yet deployed in a release, so it is expected this would not have an effect yet

Thanks @dontcallmedom  for clarifying this, wasn't sure about whether it was already deployed or not.

Thanks @pkra  and @adampage  for testing this.

**Comment by daniel-montalvo** [comment]
Moving this to draft until this is actually deployed in a respec release

**Comment by sidvishnoi** [comment]
It's released now.

**Comment by pkra** [comment]
@daniel-montalvo following today's call I checked again. I do not see any changes in the preview when forcing dark mode. 

Doe the previews need to be refreshed in some way perhaps?

File: index.html [hunk 1]
```diff
@@ -155,6 +155,7 @@
 	xref: ["dom", "accname-1.2", "core-aam-1.2", "infra", "HTML"],
 	definitionMap: []
 	};
+	darkMode: true;
```
Comment (sidvishnoi) [inline index.html:158]: We removed this config option today in favor of the color-scheme meta tag.
https://lists.w3.org/Archives/Public/spec-prod/2024AprJun/0000.html

Comment (pkra) [reply index.html:158]: Thanks @sidvishnoi !

**Comment by pkra** [comment]
@daniel-montalvo in the latest netlify preview, I still don't see a visual change when forcing darkmode (in Chrome or Firefox). Sorry :(

**Comment by kfranqueiro** [comment]
I see the change in the [preview](https://deploy-preview-2138--wai-aria.netlify.app/). My OS is set to dark mode and I see this:

screenshot of preview with light text on dark background

If I force light mode in dev tools I see the opposite:

screenshot of preview with dark text on light background

Note that the suggestion about removing `darkMode: true` from the changes still seems to be unresolved.

**Comment by pkra** [comment]
@kfranqueiro thanks for making me take another look. I now noticed that when I switch to darkmode after loading, there's no effect. But if I reload the page, it's switched to dark mode. Maybe that's the expected behavior of respec right now.

We need to fix our CSS, e.g., common.css sets table cells to a background of `#eee` which does not play well with dark mode's text color.

**Comment by pkra** [comment]
@daniel-montalvo let me know if you'd like CSS fixes as part of this PR or afterwards.

**Review by pkra — APPROVED** [review]

**Comment by daniel-montalvo** [comment]
Thanks @kfranqueiro  for taking a look at this.

@pkra  Happy for the CSS changes to be included in this PR.

**Comment by pkra** [comment]
Merging as per editors' meeting today.

**Pull request merged**